app/views/purchases/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<header id="homepage">
+  <%= link_to group_purchases_path(params[:group_id]), data: { turbo: false }  do %>
+    <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+  <% end %>
+  <h1>edit transaction</h1>
+</header>
+
+<main class="with_actions for_form">
+  <% flash.each do |name, msg| -%>
+    <%= content_tag :div, msg, class: name %>
+  <% end -%>
+  
+  <%= form_with(url: group_purchase_path, model: @purchase, method: :patch, class: "form") do |f| %>
+    <div class="field">
+      <%= f.label :name, "name" %>
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Bodyshop cream" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :amount, "amount, $" %>
+      <%= f.text_field :amount, autofocus: true, autocomplete: "amount", placeholder: "15" %>
+    </div>
+
+    <div class="checkbox">
+      <%= f.label :amount, "choose categories:" %><br />
+      <%= f.collection_check_boxes :group_ids, current_user.groups, :id, :name, :include_hidden => false %>
+      <span>No mathing category?</span>
+      <%= link_to "add a new category", new_group_path, data: { turbo: false }, class: "static"  %>
+    </div>
+
+    <div id="actions">
+      <%= f.submit "update", data: { turbo: false }, class: "btn"  %>
+    </div>
+  <% end %>
+</main>
```

Comment:
- [ ] As mentioned by the previous reviewer please make the transaction header font color white. since it is black in your design and it should be white so kindly work on changing the color 🙏🏻⭐💪🏻🎉🔥

- Your design
![image](https://github.com/yusufyusie/budget_application/assets/58553711/d9f12529-efbe-4f93-8f43-186887047868)


- Given
![image](https://github.com/yusufyusie/budget_application/assets/58553711/98a9671b-58cf-419a-8aa5-3b3b2e1adc7a)